### PR TITLE
Ansible Cleanup: Combining few tasks into one

### PR DESCRIPTION
**Pull request by abhinandanpb**
Feels good to remove code. Realised I can leverage our bundle structure to untar.

Signed-off-by: Abhinandan Prativadi <abhi@docker.com>

### CODE REVIEW

File: contrib/ansible/tasks/binaries.yaml [hunk 1]
```diff
@@ -1,52 +1,11 @@
...
-    dest: "{{ cri_release_directory }}"
-    remote_src: yes
-
-- name: "Install the containerd and cri-containerd binaries" 
-  copy:
-    src: "{{ cri_release_directory }}usr/local/bin/{{ item }}"
-    dest: "{{ local_bin_dir }}{{ item }}"
+    src: "https://storage.googleapis.com/cri-containerd-staging/cri-containerd-{{ cri_containerd_release_version }}.tar.gz"
+    dest: "/"
     mode: 0755
```
Comment (Random-Liu) [inline contrib/ansible/tasks/binaries.yaml]: What does 755 means here? Given that we have both binaries and systemd unit files inside the tarball.

File: contrib/ansible/tasks/binaries.yaml [hunk 2]
```diff
@@ -1,52 +1,11 @@
...
+    exclude: etc/cni/
```
Comment (Random-Liu) [inline contrib/ansible/tasks/binaries.yaml]: Also `/opt/cni/`.

Comment (abhinandanpb) [reply contrib/ansible/tasks/binaries.yaml]: removed both /etc/cni after your PR

**Comment by Random-Liu** [comment]
WE need to wait for https://github.com/kubernetes-incubator/cri-containerd/pulls to be merged, and update tarball version.

We could do that tomorrow morning.

File: contrib/ansible/vars/vars.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 # TODO update official versions once they are available
-cri_containerd_release_version: 0.1.0-258-g529971a
+cri_containerd_release_version: 0.1.0-271-g86ee919
```
Comment (Random-Liu) [inline contrib/ansible/vars/vars.yaml]: Get rid of the tailing space.

**Comment by Random-Liu** [comment]
LGTM with one nit.

**Pull request merged**